frigate/http.py

```diff
@@ -565,6 +565,8 @@ def events():
     has_clip = request.args.get("has_clip", type=int)
     has_snapshot = request.args.get("has_snapshot", type=int)
     include_thumbnails = request.args.get("include_thumbnails", default=1, type=int)
+    in_progress = request.args.get("in_progress", default=0, type=int)
```

Comment:
I think this behavior should match the other filters as in the default value is `None`, passing 0 will only return events not in progress and passing 1 will only return events in progress

Reply:
Right, I hadn't considered the other use, but the parameter would follow the same pattern.

Reply:
Not sure I follow what you mean by the last part?

Reply:
Sorry I dropped too many words.  I meant to say that I see your point that there's benefit of in_progress following the same pattern as has_clip and has_snapshot with possible values of none, 0, and 1.  just pushed an update that I think should work.